README.md

```diff
@@ -1,4 +1,4 @@
-Authors Haven - A Social platform for the creative at heart.
+[![Build Status](https://travis-ci.org/andela/ah-magnificent6.svg?branch=master)](https://travis-ci.org/andela/ah-magnificent6)
```

Comment:
change this to point to develop or the whole repo as we will rarely touch master branch

Reply:
@parseendavid Done